### PR TITLE
Slightly improve the Orbit STF guide

### PR DESCRIPTION
**Pull request by mahsamoosavi**

### CODE REVIEW

File: arbitrum-docs/node-running/how-tos/build-nitro-locally.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ sudo mkdir -p /usr/local/bin
 sudo ln -s  /opt/homebrew/opt/llvm/bin/wasm-ld /usr/local/bin/wasm-ld
 ```
 
-### Step 5. Configure Node [16.19](https://github.com/nvm-sh/nvm)
+### Step 5. Configure node [16.19](https://github.com/nvm-sh/nvm)
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/how-tos/build-nitro-locally.md:113]: This one should be uppercase I think, because it refers to NodeJS

**Review by TucksonDev — APPROVED** [review]
LGTM! Only added one minor comment.

File: arbitrum-docs/node-running/how-tos/build-nitro-locally.md (hunk 1 above)
Comment (mahsamoosavi) [reply arbitrum-docs/node-running/how-tos/build-nitro-locally.md:113]: Oh you're right, nvm then!

**Review by symbolpunk — APPROVED** [review]
TYSM 🔥

**Pull request merged**